src/Pmi/Evaluation/Evaluation.php

```diff
@@ -283,7 +283,7 @@ public function getForm(FormFactory $formFactory)
                 $attributes['data-parsley-lt'] = $field->max;
             }
             if (isset($field->min)) {
-                $constraints[] = new Constraints\GreaterThanEqual($field->min);
+                $constraints[] = new Constraints\GreaterThanOrEqual($field->min);
```

Comment:
This may have been a previous name of this class? Either way, I believe this was the intention.

Reply:
Looks like we've never actually had a field-level absolute minimum defined, so this has always been a bug. In the Symfony-migrated version, this is already fixed.